airbyte-integrations/connectors/source-mongodb/README.md

```diff
@@ -0,0 +1,20 @@
+# Mongodb Source 
+
+This is the repository for the Mongodb source connector, written in Ruby. 
```

Comment:
would you like to add yourself under an author header? ;)

Reply:
Why not :). But unfortunately, I did not find anything similar in other connectors. Could you please give me an example of what should I write there?

Reply:
added a suggestion below. Will review the rest of the changes later!